src/storage/src/hummock/cache.rs

```diff
@@ -924,6 +958,7 @@ mod tests {
 
             cache.release(new_entry);
             assert!((*new_entry).is_in_cache());
+            #[cfg(debug_assertions)]
             assert!((*new_entry).is_in_lru());
```

Comment:
```suggestion
            debug_assert!((*new_entry).is_in_lru());
```

I guess this will work

Reply:
It's not working if we are running `cargo bench`, where unit test will be compile while `debug_assertions` is not enabled.

The code of `is_in_lru` only exists when `debug_assertions` is enabled.
```
#[cfg(debug_assertions)]
fn is_in_lru(&self) -> bool {
    (self.flags & IN_LRU) > 0
}
```
, while the generated code of `debug_assert` is 
```
if cfg!(debug_assertions) {
 ...
}
```
, and usage for `is_in_lru` still exists even if `debug_assertions` is not enabled, and when running `cargo bench`, the compilation will faill.

Reply:
okay, I originally expected that `debug_asserts` is equivalent to `#[cfg(debug_assertions)]`.